app/domain/authentication/auth_data.rb

```diff
@@ -0,0 +1,15 @@
+
+module AuthData
+    def auth_name
+      @json_data["authenticator"]
+    end
+
+    def service_id
+      @json_data["service_id"]
+    end
+
+    def auth_parameters
```

Comment:
[NIT] - it may be worth changing these methods with `auth_` to just `parameters` or `name` as they have the "authenticator" context from the module/injected class.